enterprise/backend/test/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions_test.clj

```diff
@@ -1011,6 +1025,8 @@
                   (is (= [[1 1 14 37.65 2.07 39.72 nil "2019-02-11T21:40:27.892Z" 2 "Awesome Concrete Shoes"]]
                          (mt/rows (mt/run-mbql-query orders {:limit 1})))))))))))))
 
+;;; if this test is failing, check [[metabase.query-processor.pivot-test/drivers-test]], which is basically the same
```

Comment:
I made a copy of this test with the query as it appears post-sandboxing in `qp.pivot-test` so you don't have to run unrelated EE sandboxing tests to find out there's a bug with the Pivot QP code